src/solvers/dg/1d/containers.jl

```diff
@@ -12,17 +11,51 @@ mutable struct ElementContainer1D{RealT<:Real, NVARS, POLYDEG} <: AbstractContai
   node_coordinates::Array{RealT, 3}    # [orientation, i, elements]
   surface_flux_values::Array{RealT, 3} # [variables, direction, elements]
   cell_ids::Vector{Int}                # [elements]
+  # internal `resize!`able storage
+  _u::Vector{RealT}
+  _u_t::Vector{RealT}
+  _u_tmp2::Vector{RealT}
+  _u_tmp3::Vector{RealT}
+  _node_coordinates::Vector{RealT}
+  _surface_flux_values::Vector{RealT}
 end
 
-function Base.copy!(dst::ElementContainer1D, src::ElementContainer1D)
-  dst.u                   = src.u
-  dst.u_t                 = src.u_t
-  dst.u_tmp2              = src.u_tmp2
-  dst.u_tmp3              = src.u_tmp3
-  dst.inverse_jacobian    = src.inverse_jacobian
-  dst.node_coordinates    = src.node_coordinates
-  dst.surface_flux_values = src.surface_flux_values
-  dst.cell_ids            = src.cell_ids
+function Base.resize!(elements::ElementContainer1D{RealT, NVARS, POLYDEG},
```

Comment:
This should get at least a basic comment on what is done here (with the internal storage and the unsafe wrapping etc.).

Reply:
Done in 18e3930